docs/.vitepress/theme/components/CarbonAds.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="carbon-ads" ref="el"></div>
+  <div class="carbon-ads" ref="el" />
```

Comment:
same here

Reply:
Although these may not be necessary, it is strongly recommended for you to self close empty elements, [according to the Vue 2 Style Guide section on the topic](https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended).  The style guide is most likely applicable to Vue 3 as it has the same behavior for self closing components.  I think that the official Vue Router should probably uphold to the official Vue style guide.

Reply:
It's for components not elements. Thanks

Reply:
Sorry